backend/src/schema/types/type/Group.gql

```diff
@@ -41,6 +41,10 @@ type Group {
   myRole: GroupMemberRole # if 'null' then the current user is no member
 
   posts: [Post] @relation(name: "IN", direction: "IN")
+
+  isMutedByMe: Boolean!
+  @cypher(
+    statement: "MATCH (this)<-[m:MUTED]-(u:User {id: $cypherParams.currentUserId}) RETURN COUNT(m) >= 1")
```

Comment:
uses `EXISTS` see https://stackoverflow.com/a/42022583

Reply:
It is the same

Reply:
But better

Reply:
So let us do a refactor. This appears about ten times ...